<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeControll</class>
 <widget class="QFrame" name="RangeControll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>0:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spin_box_min">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000010000000000</double>
     </property>
     <property name="value">
      <double>0.000450000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spin_box_max">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000010000000000</double>
     </property>
     <property name="value">
      <double>0.003000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
